test/pti.jl

```diff
@@ -237,3 +237,238 @@ TESTLOG = Memento.getlogger(PowerModels)
+    #end
+
+    @testset "test case14" begin
+        file = "../test/data/pti/case14.raw"
+        test_pti_idempotent(file, PowerModels.parse_file)
+    end
+
+    # Same as the case5.raw since there is a branch with I=23, J=20. If u open it and save it from PSSE
+    # the data of this branch is I=20, J=23 but with MET = 2
+    #@testset "test case24" begin
+    #    file = "../test/data/pti/case24.raw"
+    #    test_pti_idempotent(file, PowerModels.parse_file)
+    #end
+
+    @testset "test case30" begin
+        file = "../test/data/pti/case30.raw"
+        test_pti_idempotent(file, PowerModels.parse_file)
+    end
+
+    @testset "test case73" begin
+        file = "../test/data/pti/case73.raw"
+        test_pti_idempotent(file, PowerModels.parse_file)
+    end
+
+    @testset "test frankenstein_00_2" begin
+        file = "../test/data/pti/frankenstein_00_2.raw"
+        test_pti_idempotent(file, PowerModels.parse_file)
+    end
+
+    @testset "test frankenstein_20" begin
+        file = "../test/data/pti/frankenstein_20.raw"
+        test_pti_idempotent(file, PowerModels.parse_file)
+    end
+
+    # Same as case5 and case 24
+    # See line 27 of frankenstein_70.raw
+    #@testset "test frankenstein_70" begin
+    #    file = "../test/data/pti/frankenstein_70.raw"
+    #    test_pti_idempotent(file, PowerModels.parse_file)
+    #end
+
+end
+
+
+@testset "test export all to pti" begin
+
+    function test_pti_export_all(filename::AbstractString)
+        source_data = parse_file(filename, import_all=true)
+
+        file_tmp = "../test/data/tmp.raw"
+        PowerModels.export_pti(file_tmp, source_data)
+
+        destination_data = PowerModels.parse_file(file_tmp, import_all=true)
+        rm(file_tmp)
+
+        # Delete "name" key
+        delete!(source_data, "name")
+        delete!(destination_data, "name")
+
+        @test InfrastructureModels.compare_dict(source_data, destination_data)
+    end
+
+    @testset "test case3" begin
+        file = "../test/data/pti/case3.raw"
+        test_pti_export_all(file)
+    end
+
+    @testset "test case30" begin
+        file = "../test/data/pti/case30.raw"
+        test_pti_export_all(file)
+    end
+
+    # No intentions to export INTER AREA TRANSFER
+    #@testset "test parser_test_k" begin
+    #    file = "../test/data/pti/parser_test_k.raw"
+    #    test_pti_export_all(file)
+    #end
+
+end
+
+@testset "Export File and Check Power Flows" begin
+
+    function test_pf(filename::AbstractString)
+        source_data = parse_file(filename)
+        source_pf = PowerModels.compute_ac_pf(source_data)
+
+        file_tmp = "../test/data/tmp.raw"
+        PowerModels.export_file(file_tmp, source_data)
+
+        destination_data = PowerModels.parse_file(file_tmp)
+        destination_pf = PowerModels.compute_ac_pf(destination_data)
+        rm(file_tmp)
+
+        @test InfrastructureModels.compare_dict(source_pf, destination_pf)
+    end
```

Comment:
Can you tell me about your motivation for this test?  I am inclined to say this is not required.

Reply:
Just to be sure.

It's redundant, we could remove it

Reply:
Ok, I suggest to remove it, the tests of this package already take quite some time to run.